config/default_records/plate_purposes/014_bge.wip.yml

```diff
@@ -0,0 +1,21 @@
+# Plate purposes for BGE
+---
+BGE Cherrypick:
+  type: PlatePurpose::Input
+  target_type: Plate
+  default_state: passed
+  stock_plate: false
+  cherrypickable_target: true
+  size: 96
+BGE Lib XP2:
+  target_type: Plate
+  default_state: passed
+  stock_plate: false
+  cherrypickable_target: true
```

Comment:
and only the cherrypick plate should have the cherrypickable_target true unless something special is happening here